javaparser-core/src/main/java/com/github/javaparser/JavaToken.java

```diff
@@ -110,6 +110,10 @@ public int getKind() {
         return kind;
     }
 
+    public void setKind(int kind) {
```

Comment:
Is it necessary to make the Token mutable? Could we instead instantiate a new token when we need to change the kind?